### PR TITLE
thsiswin11: Add version 1.0.50

### LINKED ISSUE #7604: [Request] ThisIsWin11
### Information

**Name:** ThisIsWin11

**Description:** ThisIsWin11 lets you customize Windows 11 all in one place. It is slowly expanded as a practical all-rounder for Windows 11, similar to Microsoft PowerToys.

**Homepage:** https://github.com/builtbybel/ThisIsWin11

**Download link(s):** https://github.com/builtbybel/ThisIsWin11/releases

**Some indication of popularity/repute:** 1.4k stars GitHub stars / https://www.youtube.com/watch?v=8ReoMuCUdKE

### PR DESCRIPTION
**Pull request by littleli**
*ThisIsWin11 lets you customize Windows 11 all in one place. It is slowly expanded as a practical all-rounder for Windows 11, similar to Microsoft PowerToys.*

Closes #7604

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by littleli** [comment]
Maybe "notes" section which informs user that this package works only on Windows 11 is a good idea?

File: bucket/thisiswin11.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.0.50",
+    "description": "The real PowerToys for Windows 11",
```
Comment (rashil2000) [inline bucket/thisiswin11.json:3]:
```suggestion
    "description": "An all-rounder tool to set up and customize Windows 11",
```

File: bucket/thisiswin11.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "homepage": "https://www.builtbybel.com/blog/19-apps/43-try-the-new-thisiswin11-app-and-get-some-powertoys-for-windows-11",
```
Comment (rashil2000) [inline bucket/thisiswin11.json:4]:
```suggestion
    "homepage": "https://github.com/builtbybel/ThisIsWin11",
```

File: bucket/thisiswin11.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "license": "MIT",
+    "url": "https://github.com/builtbybel/ThisIsWin11/releases/download/1.0.50/TIW11.zip",
+    "hash": "182f401ad57670d68e87e9970990a0b1a24679a018ce61f5dadab757e83163ca",
+    "shortcuts": [
+        [
+            "ThisIsWin11.exe",
+            "Customize Windows 11"
```
Comment (rashil2000) [inline bucket/thisiswin11.json:11]:
```suggestion
            "ThisIsWin11"
```

File: bucket/thisiswin11.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/builtbybel/ThisIsWin11"
+    },
```
Comment (rashil2000) [inline bucket/thisiswin11.json:16]:
```suggestion
    "checkver": "github",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
> Maybe "notes" section which informs user that this package works only on Windows 11 is a good idea?

I think the name is enough for that.

Also, Win11 is 64bit only, so please use architecture blocks everywhere

**Comment by littleli** [comment]
> > Maybe "notes" section which informs user that this package works only on Windows 11 is a good idea?
> 
> I think the name is enough for that.
> 
> Also, Win11 is 64bit only, so please use architecture blocks everywhere

Done. Thank you..

**Pull request merged**